### DISCUSSION TITLE
[Feature Requests] Great Swype Features

### DISCUSSION THREAD

**Discussion by adeyo50** [AnySoftKeyboard/AnySoftKeyboard#3005] [Ideas]
If anyone has ever used the original swype keyboard or swiftkey there are a few "speedy" features I feel I can no longer live without. Here they are...

1 while tapping the delete key backspaces and deletes a stroke, long pressing deletes an entire word then continuing to hold starts quickly jumping back (deleting) additional words until the key is released

2 for speedy use of the often used secondary keys (for ex ? ! , Etc) ...instead of long pressing, I simply swipe from the key to the spacebar. It's amazing how much faster this can be than long pressing

**Comment by ChaosNicro** [comment]
Additionally, bringing a Swype above the keyboard should prompt the engine to prioritize capitalized letters at that point in the Swype. While you can abuse the dictionary for capitalization, it fails with things like the German "Weg" vs. "weg", "way" vs. "away/gone".
This would likely have to be configurable,  as it would need to replace the current numbers /symbols popup.
